jwst/associations/lib/rules_level2b.py

```diff
@@ -52,6 +54,84 @@
 # --------------------------------
 # Start of the User-level rules
 # --------------------------------
+@RegistryMarker.rule
+class Asn_Lv2Coron(
+        AsnMixin_Lv2Coron,
+        DMSLevel2bBase
+):
+    """Level2b Coronagraphic Science Image Association
+
+    Characteristics:
+        - Association type: ``image2``
+        - Pipeline: ``calwebb_image2``
+        - Coron-based science exposures
+        - Single science exposure
+        - non-TSO
+        - Use rateints product as input
+        - Exclude NIRCam SW detectors that don't have occulter
+    """
+
+    def __init__(self, *args, **kwargs):
+
+        # Setup constraints
+        self.constraints = Constraint([
+            Constraint_Base(),
+            Constraint_Mode(),
+            Constraint_Image_Science(),
```

Comment:
`Constraint_Image_Science` is to match any Image-based exposure. This rule only want coronagraphic, which looks to be handled by `Constraint_Coron`, below. Should be able to remove this constraint.

Reply:
fixed in latest commit